config/samples/custom-rules/openshift-virtualization/kustomization.yaml

```diff
@@ -4,5 +4,6 @@ resources:
   - kubevirt-nonroot-feature-gate-is-enabled.yaml
   - kubevirt-no-permitted-host-devices.yaml
   - kubevirt-persistent-reservation-disabled.yaml
+  - kubevirt-no-vms-overcommiting-guest-memory
```

Comment:
We could also add it into https://github.com/ComplianceAsCode/compliance-operator/blob/master/config/samples/custom-rules/openshift-virtualization/tailored-profile.yaml so that it's run automatically.